R/01_skateboard_fiona.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: Skateboard
+output: github_document
```

Comment:
Awesome!
Can you add this?
author: Fiona

Reply:
Thanks, I added author Fiona!

Reply:
Thanks!